examples/notebooks/classification-tutorial.ipynb

```diff
@@ -46,7 +46,7 @@
    "outputs": [],
    "source": [
     "# for augmentations\n",
-    "!pip install albumentations\n",
+    "!pip install -U albumentations\n",
```

Comment:
Maybe we can just replace `!pip install catalyst`  with `!pip install catalyst[cv]` so we can control the version of albumentations by changing requirements_cv and prevent this example from failing. Also, in this case, we can remove `!pip install -U albumentations` as it is already in catalyst[cv] requirements. What do you think?

Reply:
I have some doubts about your proposal:
1. catalyst itself has other dependencies we may want to use
2. we use new functionality of albumentations (ToTensorV2), so it's better to declare explicitly about installing new version

But maybe i'm wrong, idk